index.js

```diff
@@ -205,6 +228,7 @@ class Ky {
 			// TODO: credentials can be removed when the spec change is implemented in all browsers. Context: https://www.chromestatus.com/feature/4539473312350208
 			credentials: this._input.credentials || 'same-origin',
 			...options,
+			headers: mergeHeaders(this._input instanceof globals.Request ? this._input.headers : {}, options.headers),
```

Comment:
This can be simplified to:

```suggestion
			headers: mergeHeaders(this._input.headers, options.headers),
```